standard-best-iac-structure.md

```diff
@@ -0,0 +1,167 @@
+# Creating a good IaC structure
```

Comment:
We find that using toggles is not great because when untoglling them, there's no indentation so you quickly get lost